e2e/cypress/support/commands.ts

```diff
@@ -53,6 +53,58 @@ Cypress.Commands.add('applyProjectsFilter', (projectsToFilterOn: string[]) => {
   }
 });
 
+Cypress.Commands.add('generateAdminToken', (idToken: string) => {
+  const adminTokenObj = {
+    id: 'cypress-api-test-admin-token',
+    name: 'cypress-api-test-admin-token'
+  };
+
+  // delete old token if exists
+  cy.request({
+    auth: { bearer: idToken },
+    method: 'DELETE',
+    url: `/apis/iam/v2beta/tokens/${adminTokenObj.id}`,
+    failOnStatusCode: false
+  }).then((resp: Cypress.ObjectLike) => {
+    expect([200, 404]).to.include(resp.status);
```

Comment:
Reading cypress tests recently I noticed we have been doing this wrong.
It should really be "expect _actual value_ to satisfy _expected value_", which means:
```suggestion
    expect(resp.status).to.be.oneOf([200, 404]);
```
Not suggesting you fix all of the occurrences in this PR, but better to start using the correct version. 😁

Reply:
good point! 👍